cloudformation.yaml

```diff
@@ -113,3 +113,12 @@ Resources:
             Timeout: 100
         DependsOn: LambdaRole
 
+    Scheduler:
+        Type: AWS::Events::Rule
```

Comment:
Do we need to give the scheduler permission to invoke the lambda? Something like this?
```yaml
  SchedulerLambdaPermission:
    Type: AWS::Lambda::Permission
    Properties:
      Action: lambda:InvokeFunction
      FunctionName: !GetAtt UserReminderLambda.Arn
      Principal: events.amazonaws.com
      SourceArn: !GetAtt Scheduler.Arn
```

Reply:
Yeah, looks like it is needed. What a waste of time and energy 💩

Reply:
The Serverless Transform does it automatically 😉

Reply:
Done, thanks for your advice 👍

Reply:
Ooooh, so I might take up on your offer and do it in a separate PR (I did not see the advantage but if it's cleaner and easier to reason about, then 👍 )